### PR TITLE
Edits for dose prediction (regression): Ensure that segmentation mask still works.

### PR DESCRIPTION
**Pull request by doucejosy**
Edited analyze, preprocess, train, evaluate functions. More needed: train/val losses, dose metrics, full preprocessing of input data for dose prediction. I made sure that both segmentation mask and dose prediction still work with these changes.

The assumption is that the input data for now are already preprocessed. I can always integrate the preprocessing steps later as the main task is to see if we can train and get reasonable results with these changes.
- analyze.py and preprocess.py are edited to create config file and numpy input and target data for dose prediction.
- training function is edited to include train/val loss such as MSE, MAE, .... For now when training, it seems like the val loss doesn't really do anything, so I am investigating that. 
- Inference code is edited to ensure that we can predict dose data.
- Evaluation code still needs major overahuls. For now, I am using dice/haus95, just because I wanted to add basic changes that will allow the code to work with dose data. Pretty soon I will also include correct dose metrics such as dose score, dvh, ...

I still have many questions regarding the code and we can schedule a meeting for Friday to discuss them.

### CODE REVIEW

**Comment by doucejosy** [comment]
New commit: I just edited the loss and loss_utils code to correctly compute training and loss function. I wasn't indexing the y_true and y_pred correctly.

**Comment by aecelaya** [comment]
Closing pull request since we moved this to another repo.
